src/navigation.tpl

```diff
@@ -1,16 +1,10 @@
  <ul>
     <li><a href="about-juju.html">About Juju</a></li>
-    <li><a href="introducing-2.html">Introducing Juju 2.0!</a></li>
-    <li><a href="whats-new.html">What's new in 2.1</a></li>
 <!-- SECTION -->
-    <li class="section"><h4 class="header toggle-target">Get Started</h4>
+    <li class="section"><h4 class="header toggle-target">Quick Guides</h4>
```

Comment:
I guess I've been in too much "getting started" all over the place and as we're starting to refer to things as tutorials or guides I thought it might be a good update to reduce using "getting" so much any more. I'm +1 on moving the First use to the new "Get started". I almost did that in here.